app/models/container_image.rb

```diff
@@ -97,5 +97,10 @@ def disconnect_inv
     save
   end
 
+  def self.disconnect_inv(ids)
+    _log.info "Disconnecting ContainerImages ids [#{ids}]"
```

Comment:
right, I don't have the EMS object here. Not sure if I should pass it for the purpose of logging?